xbmc/cores/VideoPlayer/VideoRenderers/OverlayRendererDX.cpp

```diff
@@ -248,8 +248,11 @@ COverlayImageDX::COverlayImageDX(CDVDOverlayImage* o)
     else
 #endif
     {
-      /* render aligned to screen to avoid cropping problems */
-      m_align  = ALIGN_SCREEN;
+      CRect view = CServiceBroker::GetWinSystem()->GetGfxContext().GetViewWindow();
```

Comment:
this is wrong for multiple reasons:

1) Windows only
2) don't call out to gfx context, renderer already knows viewRect
3) won't work in videoControl
4) won't work with overscan set